components/landing/TheQiskitCapabilitiesSection/index.vue

```diff
@@ -1,16 +1,233 @@
+        class="qiskit-capabilities-section__copy-button"
+        @click="[copyToClipboard(), $trackClickEvent(copyCodeCTA.segment)]"
+      >
+        <span class="qiskit-capabilities-section__copy-button__label">{{ copyCodeCTA.label }}</span>
+        <Copy16 class="qiskit-capabilities-section__copy-button__icon" />
+      </cv-button>
+    </div>
   </article>
 </template>
 
 <script lang="ts">
-import Vue from 'vue'
-import { Component } from 'vue-property-decorator'
+import { Component, Mixins } from 'vue-property-decorator'
+import ScrollSectionsMixin from '~/mixins/scrollBetweenSections'
+
+interface QiskitCapability {
+  /** The visible title of the feature */
+  title: string
+  /** The visible description of the feature */
+  description: string
+  /** The image of the feature */
+  thumbnailResource: string
+}
 
 @Component
-export default class TheQiskitCapabilitiesSection extends Vue {
-  activeBlocks = [true, false, false]
+export default class TheQiskitCapabilitiesSection extends Mixins(ScrollSectionsMixin) {
+  capabilities = [
+    {
+      title: 'Circuit Library',
+      description: 'Qiskit includes a comprehensive set of quantum gates and a variety of pre-built circuits so users at all levels can use Qiskit for research and application development.',
+      thumbnailResource: '/images/landing-page/feature-circuit.png',
+      link: {
+        url: 'https://qiskit.org/documentation/apidoc/circuit_library.html',
+        label: 'Learn more',
+        segment: { cta: 'feature-circuit', location: 'homepage-capabilities-section' }
+      }
+    },
+    {
+      title: 'Transpiler',
+      description: 'The transpiler translates Qiskit code into an optimized circuit using a backend’s native gate set, allowing users to program for any quantum processor or processor architecture with minimal inputs.',
+      thumbnailResource: '/images/landing-page/transpiler.png',
+      link: {
+        url: 'https://qiskit.org/documentation/apidoc/transpiler.html',
+        label: 'Learn more',
+        segment: { cta: 'transpiler', location: 'homepage-capabilities-section' }
+      }
+    },
+    {
+      title: 'Run on real hardware',
+      description: 'Users can run and schedule jobs on real quantum processors, and employ Qiskit Runtime to orchestrate quantum programs on cloud-based CPUs, QPUs, and GPUs.',
+      thumbnailResource: '/images/library/chip-01.png',
+      link: {
+        url: 'https://qiskit.org/documentation/apidoc/ibmq_provider.html',
+        label: 'Learn more',
+        segment: { cta: 'run-on-hardware', location: 'homepage-capabilities-section' }
+      }
+    },
+    {
+      title: 'Try it yourself',
+      description: 'Ready to explore Qiskit’s capabilities for yourself? Copy the code to the right of this paragraph and try running it in your local Python environment. You can also click the link to the IBM Quantum Lab and test it there.',
+      thumbnailResource: '/images/landing-page/med_02_1.png'
+    }
```

Comment:
I this this is missing the `segment` attribute.

We should probably add it to the interface.